Makefile

```diff
@@ -19,6 +19,8 @@ test:
 	make verify && make unit-test
 
 unit-test:
+	export MEMBQL_API_ENDPOINT_PROD=1; \
```

Comment:
Why reset these?

Reply:
Because they are needed in some form for the unit tests to pass (on circle) and this is how `signup` does it

Reply:
The functions are stubbed in the unit tests but the library requires those env properties are set.

Reply:
Shouldn't they be in the generated .env anyway?